src/Angular.js

```diff
@@ -1147,7 +1177,7 @@ function snake_case(name, separator){
 
 function bindJQuery() {
   // bind to jQuery if present;
-  jQuery = window.jQuery;
+  jQuery = jq() ? window[jq()] : window.jQuery;
```

Comment:
@lgalfaso hm, guess I need to change my unit tests then.  Thanks for the heads up.  Will change right now.